### PR TITLE
Show undecided matches greyed out in dashboard

### PR DESCRIPTION
**Pull request by elp49**

### CODE REVIEW

File: frontend/src/components/MailItem.tsx [hunk 1]
```diff
@@ -16,6 +16,11 @@ const useStyles = makeStyles({
   mailItem: {
     minHeight: "4rem",
     padding: "0 1rem",
+    marginBottom: "5px",
+  },
+  undecidedMatch: {
+    backgroundColor: "grey",
```
Comment (mneedleman) [inline frontend/src/components/MailItem.tsx]: It's a little dark, maybe a lighter grey?

**Review by mneedleman — CHANGES_REQUESTED** [review]
One tiny thing

File: frontend/src/components/MailItem.tsx (hunk 1 above)
Comment (jcarrete5) [reply frontend/src/components/MailItem.tsx]: I agree. Looks a bit too dark.

**Review by rao53 — COMMENTED** [review]
No issues here for me and could be merged after the color changes requested.

**Review by jcarrete5 — APPROVED** [review]

File: frontend/src/components/MailItem.tsx [hunk 2]
```diff
@@ -59,7 +66,7 @@ export default function MailItem({
       direction="row"
       justify="space-between"
       alignItems="center"
-      className={mailItem}
+      className={`${mailItem} ${!isDecided && undecidedMatch}`}
```
Comment (mneedleman) [inline frontend/src/components/MailItem.tsx:69]: We're toeing the line between doing it ourselves and wanting to do it cleanly. If we need more logic than this for class names, there's a `classnames` library that will help make this clean. Just food for thought

**Review by mneedleman — APPROVED** [review]

**Review by rao53 — APPROVED** [review]

**Pull request merged**